modules/administration-guide/partials/proc_enabling-and-exposing-che-metrics.adoc

```diff
@@ -19,7 +19,7 @@ spec:
 +
 [NOTE]
 ====
-When {prod} is installed from the OperatorHub using the default `CheCluster` CR, `spec.metrics.enable` is set to `true` by default. For example:
+When {prod} is installed from the Red Hat Ecosystem Catalog using the default `CheCluster` CR, `spec.metrics.enable` is set to `true` by default. For example:
```

Comment:
```suggestion
Installing {prod} from the Red Hat Ecosystem Catalog by using the default `CheCluster` CR sets the `spec.metrics.enable` to `true`. For example:
```

Reply:
If you want to emphasize that using the default `CheCluster` CR produces this outcome, as opposed to the outcome from using other possible installation methods, then move it up to the front of the sentence:
```suggestion
When {prod} is installed from the Red Hat Ecosystem Catalog using the default `CheCluster` CR, `spec.metrics.enable` is set to `true` by default. For example:
```
Using the default `CheCluster` CR to install {prod} from the Red Hat Ecosystem Catalog sets the `spec.metrics.enable` parameter to `true`.
```

Reply:
The when clause became irrelevant.